File: src/main/java/com/github/graqr/threshr/model/redsky/products/pdp/client/RatingsAndReviews.java
package com.github.graqr.threshr.model.redsky.products.pdp.client;

import com.fasterxml.jackson.annotation.JsonProperty;
import io.micronaut.serde.annotation.Serdeable;

import java.util.List;

@Serdeable
public record RatingsAndReviews(
        @JsonProperty("has_verified") Boolean hasVerified, Statistics statistics,
        @JsonProperty("most_recent") List<MostRecent> mostRecent, List<String> photos) {
}

File: src/main/kotlin/com/graqr/threshr/circleStore/model/product/RatingsAndReviews.kt
package com.graqr.threshr.circleStore.model.product

import com.fasterxml.jackson.annotation.JsonProperty
import io.micronaut.serde.annotation.Serdeable

@Serdeable
data class RatingsAndReviews(
    @param:JsonProperty("has_verified") val hasVerified: Boolean,
    val statistics: Statistics,
    @param:JsonProperty("most_recent") val mostRecent: List<MostRecent>,
    val photos: List<String>
) {}
